analysis_rsp.tex

```diff
@@ -0,0 +1,50 @@
+
+\subsubsection{Portal}
+
+\textbf{P-101}: User interface cone search on the object catalog, review the results overplotted on the HIPS map.
+
+\textbf{P-102}: Astronomical Data Query Language (ADQL) cone search on the object catalog with magnitude constraints, create a color-magnitude diagram in the results view.
+
+\textbf{P-103}: Create exploratory color plots of different trans-Neptunian object (TNO) populations and perform statistical analyses to determine potential correlations. 
+
+\textbf{P-201}: Perform an LSST light curve classification (e.g., stars vs quasars vs AGNs) using Portal tools, leveraging visualization features to distinguish populations.
+
+\textbf{P-301}: Query annual data releases and deepCoadds for overlaps with other datasets, and generate publication-quality plots of data and images.
+
+\textbf{P-302}: Query survey property maps and cross-correlate them with shape and photo-z catalogs to investigate cosmic shear. Perform advanced filtering to optimize query performance.
+
+\subsubsection{Notebook}
```

Comment:
Review and revise the NB and API analysis use cases in the same way as Portal (make more generic, more functionality-focused, not like science use cases; ok to keep a 'for example...' illustration here and there when needed)